libs/chiliui/chili/headers/skinutils.lua

```diff
@@ -613,7 +613,7 @@ function DrawEditBox(obj)
 	local font = _GetControlFont(obj)
 	local displayHint = false
 
-	if text == "" and not obj.state.focused then
+	if text == "" then
```

Comment:
I can see this being controversial, but I think it's better to keep the hint text until the user has typed something. This depends on the hint text having an obviously different format / syntax to what the user would type.